include/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.hpp

```diff
@@ -145,6 +147,8 @@ class Propagator
     /// @endcode
     void clearDynamics();
 
+    void addManeuvers(const Array<Maneuver>& aManeuverArray);
```

Comment:
Should add a method to get/access maneuvers as well.

Reply:
So actually I don't think we should do this because the logic to extract a maneuver from a `propagator` will be pretty complicated because the Maneuver object isn't stored as a memvar inside of `Propagator` (it should stay that way), so the logic would have to decide what `TabulatedDynamics` is "considered" a maneuver and what isn't (cause of course `TabulatedDynamics` don't just have to be maneuvers.